### PR TITLE
Vimundad appgwt: Adding Rule Priority for Application Gateway RequestRoutingRule

### PR DESCRIPTION
**Pull request by vinaymundada27**
## Description
Adding a parameter 'Priority' to the Application Gateway RequestRoutingRule object.
Swagger PR - https://github.com/Azure/azure-rest-api-specs/pull/7131

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Network/Network/help/Set-AzApplicationGatewaySslCertificate.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ The **Set-AzApplicationGatewaySslCertificate** cmdlet updates an SSL certificate
 ### Example 1: Update an existing SSL certificate on Application Gateway
 ```
 PS C:\> $appGW = Get-AzApplicationGateway -Name "ApplicationGateway01" -ResourceGroupName "ResourceGroup01"
+[//]: # ([SuppressMessage\("Microsoft.Security", "CS002:SecretInNextLine", Justification="Test password for the ssl certificate"\)])
```
Comment (msJinLei) [inline src/Network/Network/help/Set-AzApplicationGatewaySslCertificate.md:30]: It is better a use a variable in the replacement of "P@ssw0rd"

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/Network/Network/help/Set-AzApplicationGatewaySslCertificate.md [hunk 2]
```diff
@@ -27,7 +27,8 @@ The **Set-AzApplicationGatewaySslCertificate** cmdlet updates an SSL certificate
 ### Example 1: Update an existing SSL certificate on Application Gateway
 ```
 PS C:\> $appGW = Get-AzApplicationGateway -Name "ApplicationGateway01" -ResourceGroupName "ResourceGroup01"
-PS C:\> $password = ConvertTo-SecureString "P@ssw0rd" -AsPlainText -Force
+PS C:\> $passwordString = "P@ssw0rd"
```
Comment (msJinLei) [inline src/Network/Network/help/Set-AzApplicationGatewaySslCertificate.md:30]: what I mean is not to list password content in the doc, but use a variable as the replacement such as

PS C:\> $passwordPlainString
PS C:\> $password = ConvertTo-SecureString $passwordPlainString -AsPlainText -Force

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/Network/Network/help/Set-AzApplicationGatewaySslCertificate.md (hunk 2 above)
Comment (vinaymundada27) [reply src/Network/Network/help/Set-AzApplicationGatewaySslCertificate.md:30]: And no need to declare the `$passwordPlainString` variable as well right?

**Review by msJinLei — APPROVED** [review]

**Pull request merged**